client/utils/navigation.js

```diff
@@ -1,6 +1,4 @@
 import { history } from 'base'
 
 // eslint-disable-next-line import/prefer-default-export
-export function navigateTo(name) {
-  return history.push({ name })
-}
+export const navigateTo = history.push
```

Comment:
I mean `export const navigateTo = history.push.bind(history)`.  Not sure if it's needed, but if we have a component that does `onClick={() => navigateTo('someRoute')}` would that be an issue if not bound?